css/ampshared.css

```diff
@@ -229,7 +231,9 @@ i-amphtml-sizer {
 }
 
 .i-amphtml-notbuilt,
-[layout]:not(.i-amphtml-element)
+[layout]:not(.i-amphtml-element),
+[width][height][sizes]:not([layout]):not(.i-amphtml-element),
+[width][height][heights]:not([layout]):not(.i-amphtml-element)
```

Comment:
We're not handling any of the other implicit layouts either. Not sure if it's better to only fix this for implicit responsive or leave them all unfixed.

Consider leaving a note/TODO about this at least.

Reply:
`fixed`, `fixed-height`, `fluid` and `container` are the missing ones. `fluid` isn't documented on https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/#layout, so lets ignore it.

`fixed` and `fixed-height` are implicit based on `height` and `width` only. Unfortunately, that's very generic, and there are non-amp elmeents that fit that:

- width + height
  - amp-script > canvas
  - SVG > *
  - td
  - th
  - input
- width
  - table

I'm not sure we want to apply these styles to those elements.

I'll add a comment.

Reply:
Ok, thanks for looking into it.